gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc

```diff
@@ -1,47 +1,51 @@
 //tag::intro1[]
-Using encryption, Transport Layer Security (TLS) policies protect communication between clients, API Instances, and upstream services. To use TLS when creating an API Instance using Flex Gateway in Local Mode as your runtime, you must configure the TLS context in a YAML configuration file.
+Using encryption, Transport Layer Security (TLS) policies protect communication between clients, API instances, and upstream services. To use TLS when creating an API instance using Flex Gateway in Local Mode as your runtime, you must configure the TLS context in a `PolicyBinding` YAML configuration file.
+
+When you add a TLS context, you can also select the ciphers to use with the TLS context and configure different inbound and outbound validation settings.
 
 //end::intro1[]
+
 //tag::intro2[]
-You can format the YAML file to configure TLS either for a specific API Instance
-or or for all API Instances running on your Flex Gateway.
+Flex Gateway supports the following:
+
```

Comment:
Suggest adding periods for the bullet items.

Reply:
I'm not sure on this. The items themselves are fragments, but they complete the sentence. I don't think they should periods though.

From the style guide:

Follow these style guidelines when creating your bullet list:

- Limit list hierarchy to two levels: three at most. Check with your editor if you need help doing this.
- Capitalize the first letter of each item. Rework to not begin list items with terms that are case-sensitive.
- Make list items parallel. For example, make all list items a noun or all list items a phrase that starts with a verb.
- End list items with periods, unless the list is composed entirely of fragments,